test/verify/check-superuser

```diff
@@ -65,14 +65,11 @@ class TestSuperuser(testlib.MachineCase):
         b.drop_superuser()
 
         # We want to be lectured again
-        self.restore_file("/var/db/sudo/lectured/admin")
-        self.restore_file("/var/lib/sudo/lectured/admin")
-        m.execute("rm -rf /var/{db,lib}/sudo/lectured/admin")
+        self.restore_dir("/var/db/sudo/lectured")
```

Comment:
The old code also had /var/lib/sudo/lectured , but looking at tests that doesn't seem to be necessary any more.